palm/plugins/dbt/commands/cmd_dbt-config.py

```diff
@@ -0,0 +1,29 @@
+import click
+from pathlib import Path
+from palm.plugins.dbt.plugin_config import PluginConfig
+
+
+@click.command("dbt-config")
```

Comment:
@jakeberesford-palmetto question... what if it were just called *config* 🤔

I would love for it to be run as `palm dbt config` which feels so much cleaner, the issue here is that `palm dbt` is going to become the passthrough command, and not a reference to the dbt plugin itself. Also, `config` alone would be overwritten/conflicted with other plugins' `config` commands or local config commands.

Reply:
I like `config` :)

Reply:
But what about multiple plugins with `config` commands?

Reply:
That gets gnarly... depends on the plugin load order, the last loaded would be the one to run... not great.

We probably want to think about adding `PluginConfig` as a feature of `BasePlugin` and designing a solution that will work well for generating the config when multiple plugins are installed, possibly adding a `config` subcommand to the `cmd_plugin` command group in palm-cli.

Let's leave this as `dbt-config` for now. We can work on the palm-cli updates to support this and update later.